rest_framework/schemas/openapi.py

```diff
@@ -258,6 +258,9 @@ def _map_field(self, field):
         if isinstance(field, serializers.ListField):
             return {
                 'type': 'array',
+                'items': {
+                    "type": self._map_field(field.child).get('type')
+                },
```

Comment:
Should we only be including `items` if `child` is not `None`?

Reply:
if `child` is `None` the type will be `string`.

`items` is required, otherwise the component would not be able to be rendered

Reply:
If `child` is `None` then any type is valid.
In JSON Schema you'd usally omit `items` in that case (or use `"items": {}`)

Reply:
Makes sense.
Since `ListField` child is `_UnvalidatedField` if omitted, should I just check the following?
`if not isinstance(field.child, _UnvalidatedField):`